release/long_running_tests/workloads/serve_failure.py

```diff
@@ -9,24 +9,24 @@
 from ray.cluster_utils import Cluster
 
 num_redis_shards = 1
-redis_max_memory = 10**8
-object_store_memory = 10**8
-num_nodes = 5
-cpus_per_node = 2
+redis_max_memory = 10 ** 8
+object_store_memory = 10 ** 8
+num_nodes = 1
```

Comment:
serve is having some trouble with multiple clusters on the same node https://github.com/ray-project/ray/issues/12779